Enhancements for op-build ci.

pridhiviraj opened the pull request:
This patch adds support for running ci for individual platforms,
as well as it can generate firmware images for all the platforms.

For running ci to individual platforms:
DOCKER_PREFIX=sudo  ./ci/build.sh -p palmetto
DOCKER_PREFIX=sudo  ./ci/build.sh -p garrison

For running ci to all the platforms:
DOCKER_PREFIX=sudo  ./ci/build.sh -a
DOCKER_PREFIX=sudo  ./ci/build.sh

And also it contains missing packages required for fedora23 Dockerfile.

Signed-off-by: Pridhiviraj Paidipeddi <ppaidipe@linux.vnet.ibm.com>

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/open-power/op-build/580)
<!-- Reviewable:end -->

op-jenkins commented:
Can one of the admins verify this patch?

dcrowell77 commented:
@jazurin should take a look

stewart-ibm commented on ci/build.sh:
```diff
@@ -1,5 +1,36 @@
+  case $opt in
+    a)
+      echo "Build firmware images for all platforms"
+      TEMP=""
```
Instead of TEMP, it'd be better to use PLATFORMS or PLATFORM, as TEMP could be confused with a temporary directory

pridhiviraj commented:
In this version V2 did below changes:
         Renamed TEMP with PLATFORMS
         changed -p option to List of comma separated platform names to be used for building the fw images.

jazurin commented:
ok to test

williamspatrick commented:
I do not feel competent enough to review this.  Need reviews from @geissonator / @stewart-ibm .

op-jenkins commented:
Can one of the admins verify this patch?

op-jenkins commented:
Can one of the admins verify this patch?

stewart-ibm commented:
This looks good to me. We should probably bump fedora23 to 24 too.
